gems/quilt_rails/lib/quilt_rails/trusted_ui_server_csrf_strategy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Quilt
+  class TrustedUiServerCsrfStrategy
```

Comment:
Could we instead inherit from `NullSession` here and just call `super` in `handle_unverified_request`?

Reply:
If we're going with `Exception`, we could inherit from that instead.

Reply:
I feel like that's stretching OO for minor benefit.  It would create an, `is one thing and another thing` relationship, rather than an `is a`.

That said, I'm not a Rails developer.  If you feel that inheritance would be more idiomatic, I can be easily swayed 😄

Reply:
Yeah, that sounds fine to me :D I wish it was a concern so we wouldn't have to instantiate another complete object, but I doubt it makes a substantial difference.